applications/crossbar/src/crossbar_maintenance.erl

```diff
@@ -435,13 +497,13 @@ validate_account(JObj, Context) ->
                                   ,{fun cb_context:set_resp_status/2, 'fatal'}
                                   ])
               ],
-    Context1 = crossbar_bindings:fold(<<"v1_resource.validate.accounts">>, Payload),
+    Context1 = crossbar_bindings:fold(<<"v2_resource.validate.accounts">>, Payload),
```

Comment:
There is no diff in /v1/accounts and /v2/accounts, this just makes sure if there was a split, that we would use /v2